### ISSUE TITLE
Missing ClearType for Japanese text

### ISSUE THREAD

**Issue by SlimDread** [RPCS3/rpcs3#12062]
Japanese text seems to be missing ClearType and as a result appears aliased when navigating the game and trophy menu.
RPCS3 version: 0.0.22-13635-7c8fbc35 Alpha
Specs:
Windows 10
i5-8300H
GTX 1050
512.15

How the title bar looks like with ClearType:
![](https://cdn.discordapp.com/attachments/277227681836302338/977850191610777640/unknown.png)

How it looks like on RPCS3 without ClearType:
![](https://cdn.discordapp.com/attachments/319224795785068545/977855860850184272/unknown.png)
![](https://cdn.discordapp.com/attachments/319224795785068545/977856864446464051/unknown.png)

English text for reference:
![](https://cdn.discordapp.com/attachments/319224795785068545/977857337727537182/unknown.png)
![](https://cdn.discordapp.com/attachments/319224795785068545/977859515187855390/unknown.png)

**Log**
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/8749101/RPCS3.log)

**Comment by AniLeo** [comment]
Missing screenshots, need updated test, specially now that we use a different Qt version

**Comment by digant73** [comment]
@SlimDread is that still an issue?

**Comment by 13xforever** [comment]
Yes, nothing has changed. At least on Windows the font substitution fallback is picking up MS Gothic font, which doesn't have ClearType hinting. IDK if it's possible to configure multiple fonts with fallback in QT, but having Yu Gothic instead would solve the issue.

<img width="3840" height="2088" alt="Image" src="https://github.com/user-attachments/assets/a2f27d3b-8586-4688-b0f1-431baa6493b4" />
<img width="2625" height="1559" alt="Image" src="https://github.com/user-attachments/assets/2fbac449-c135-407c-a629-c46ebadccd1d" />
